BraveRewardsUI/Settings/Wallet/WalletDisclaimerView.swift

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+
+class WalletDisclaimerView: UIView {
+  
+  var labels: [LinkLabel] = [] {
+    willSet {
+      labels.forEach { $0.removeFromSuperview() }
+    }
+    didSet {
+      labels.forEach {
+        stackView.addArrangedSubview($0)
+      }
+    }
+  }
+  
+  private let stackView = UIStackView().then {
+    $0.axis = .vertical
+    $0.spacing = 10
+  }
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+  
+    addSubview(stackView)
+    
+    stackView.snp.makeConstraints {
+      $0.edges.equalTo(self).inset(8)
+    }
+    
+    backgroundColor = UIColor(white: 0.0, alpha: 0.04)
```

Comment:
Could you check if this looks good on iOS 12 with light and dark theme?
If not we would need to change this to `appearanceBackgroundColor` I believe

Reply:
It seemed to work? 
![Simulator Screen Shot - iPhone Xs - 2019-10-22 at 12 00 17](https://user-images.githubusercontent.com/529104/67305693-c278af80-f4c3-11e9-8d67-ba20f6eb476a.png)

Maybe I should change it to `appearanceBackgroundColor` anyways?

Reply:
no, leave it as it is, we do not use this property in specific classes only in appearance proxies from what I see, theming is gonna drag us down a bit until we drop iOS 12 support